### PR TITLE
2024a r-sync

### PR DESCRIPTION
**Pull request by atheo89**
fixes: https://github.com/red-hat-data-services/notebooks/pull/366

### CODE REVIEW

File: jupyter/tensorflow/ubi9-python-3.9/Pipfile.lock [hunk 1]
```diff
@@ -1054,19 +1087,24 @@
...
-                "sha256:72fd4733b80b6d777dcde515628a9eb4a577339437012874ea286bca7261ee65",
-                "sha256:8eb87396435c19b20d32abd2f984e31c191a15284af72eb922f10e5bde9c04cc"
+                "sha256:25df55f327ef021de8be50bad0dfd4a916ad0de96da86cd05661c9297723ad3f",
+                "sha256:f4c64ed4e01e8e8b646ef34c018f8bf3338df0c8e37d8b3bba40e7f574a3278a"
             ],
+<<<<<<< HEAD
             "markers": "python_full_version >= '3.7.0'",
             "version": "==2.34.0"
+=======
+            "markers": "python_version >= '3.7'",
```
Comment (harshad16) [inline jupyter/tensorflow/ubi9-python-3.9/Pipfile.lock:1105]: some conflicts have been missed in these files

**Review by harshad16 — CHANGES_REQUESTED** [review]
Some conflict still pending

**Review by harshad16 — CHANGES_REQUESTED** [review]
Another requirement 
we had remove the files based on this PR: https://github.com/red-hat-data-services/notebooks/pull/367
please remove those files from this PR.

**Comment by atheo89** [comment]
Good catch, all set

**Comment by harshad16** [comment]
/lgtm
/approve

thanks :+1:

**Pull request merged**